ksql-rest-app/src/test/java/io/confluent/ksql/rest/integration/PullQueryFunctionalTest.java

```diff
@@ -91,7 +91,7 @@
   private static final Format VALUE_FORMAT = FormatFactory.JSON;
   private static final int HEADER = 1;
 
-  private static final TestBasicJaasConfig JASS_CONFIG = TestBasicJaasConfig
+  private static final TestBasicJaasConfig JAAS_CONFIG = TestBasicJaasConfig
```

Comment:
Not related to your PR.. but is PullQueryFunctionalTest the only integration test which tests with actual Jaas config? That worries me slightly!

Reply:
Nope, there's also https://github.com/confluentinc/ksql/blob/master/ksql-cli/src/test/java/io/confluent/ksql/cli/BasicAuthFunctionalTest.java. Not sure why creation of the JAAS config isn't standardized among integration tests. Probably worth cleaning up at some point.